dm/dumpling/dumpling.go

```diff
@@ -193,10 +199,26 @@ func (m *Dumpling) Update(context.Context, *config.SubTaskConfig) error {
 	return nil
 }
 
+var mu sync.Mutex
```

Comment:
this variable should belongs to `m *Dumpling`

Reply:
I understand that this variable is one of `m *Dumpling` fields, so I should add a new field to `*Dumpling`